text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
 
 # Summary
 
-This RFC proposes creating a RustConf steering committee to represent the Rust Project’s perspectives and priorities throughout the RustConf 2024 planning process. 
+This RFC proposes creating an advisory Rust Steering Committee for RustConf to represent Rust Project and community perspectives and priorities throughout the RustConf 2024 planning process. 
 
-# Proposal
-
-The Rust Foundation staff team would like to propose that a RustConf steering committee composed of ~5 Rust Project members be formed with an agreed-upon process of member appointment and scope established.
+# Definitions
 
-A Rust Project steering committee for RustConf could be defined as:
+- "Steering Committee": A group of advisors that present recommendations to an organizing group (in this case, the "organizing group" being the Rust Foundation as the organizer of RustConf 2024). Often, a steering committee will be in place to represent the interests of a specific group (in this case, the Rust Project and community)
+- "Program Committee": The group responsible for selecting speakers/talks to appear on the RustConf agenda/"program". The Program Committee for RustConf has historically been independent of the organizer and composed largely of members of the Rust Project. The Foundation intends to maintain this tradition in 2024 but will provide oversight and facilitation.
+- "CFP": Stands for "Call for Proposals". This is the period and process during which we will solicit talk submissions for RustConf. The Program Committee will select submissions from the CFP to appear on the program and might also advocate for inviting several speakers who did not submit a talk. 
+  
+# Proposal
 
-A group of ~5 volunteers from within the Leadership Council and the Project Directors responsible for providing input and direction to the Rust Foundation staff team regarding RustConf 2024. The Rust Foundation will lean on the steering committee to advocate for areas of desired involvement besides the traditional UnConference Day and Program Committee. 
+The Rust Foundation staff team is proposing the formation of a RustConf Steering Committee with the following parameters:
+- Members: 4 total members from the Rust Project -- one leadership council member, one Project Director member, and two members from the wider Rust community. We feel that an even split between leadership members and community members makes sense as the former positions can provide context and Project authority while the latter can advocate for those who are not necessarily in a position of power within the Project. We will lean on the Leadership Council as a whole to select this group based on a process of self-nomination initiated by the Foundation and promoted by the Leadership Council.  
+- Scope: The Foundation would like the Steering Committee's involvement in the following areas:
```

Comment:
Nit: I doesn't strike me as normal for an RFC to talk about the desires of the author unless that information is relevant for justifying why the authors have chosen a certain way between tradeoffs or if the authors would like to throw in a "nice-to-have" loose requirement. For more foundational information, a more direct wording is preferred: 

```suggestion
- Scope: The Steering Committee's involvement will be in the following areas:
```